opentracing/src/main/java/opentracing/propagation/SplitTextFormatInjector.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package opentracing.propagation;
+
+import java.util.Map;
+
+/**
+ *
+ */
+interface SplitTextFormatInjector extends Injector<Map> {
```

Comment:
> will not be able to distinguish the "tracer state" from the "baggage" unless we document the TRACE_* prefix strings and expose them publicly.

that was exactly my point below and in the other PR with actual injectors impl. In order to be a "split" carrier, it needs to have two maps, otherwise end user cannot distinguish which items in the map are which. Sharing the prefix strings is not an option because the prefixes _must_ be different for different tracers (the `B3_xxx` example). The two maps should be the final, ready for wire representation, the end user should not need to peek into them.